drivers/eaton-mib.c

```diff
@@ -237,10 +245,111 @@ static info_lkp_t outlet_switchability_info[] = {
 	{ 0, NULL }
 };
 
+static info_lkp_t marlin_ambient_presence_info[] = {
+	{ -1, "unknown" },
+	{ 0, "no" },  /* disconnected */
+	{ 1, "yes" }, /* connected */
+	{ 0, NULL }
+};
+
+static info_lkp_t marlin_threshold_status_info[] = {
+	{ 0, "good" },          /* No threshold trigged */
+	{ 1, "warning-low" },   /* Warning low threshold trigged */
+	{ 2, "cricital-low" },  /* Critical low threshold trigged */
+	{ 3, "warning-high" },  /* Warning high threshold trigged */
+	{ 4, "cricital-high" }, /* Critical high threshold trigged */
+	{ 0, NULL }
+};
+
+static info_lkp_t marlin_threshold_frequency_status_info[] = {
+	{ 0, "good" },          /* No threshold trigged */
+	{ 1, "out of range" },  /* Frequency out of range trigged */
```

Comment:
theoretically, there is no standard defined for such things. Historically, we used to have single-word values for such things. Practically, the other threshold status being dash-separated, I've also applied the same to the "out-of-range".
Fixed, thanks